src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1913 @@
+                _currentLanguageChanged?.Invoke(null, args);
+            };
+                ErrorCode error = VcMgrSetCurrentLanguageChangedCb(_languageDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add CurrentLanguageChanged Failed with error " + error);
+                }
+
+                else
+                {
+                    _currentLanguageChanged += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetCurrentLanguageChangedCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove CurrentLanguageChanged Failed with error " + error);
+                }
+
+                _currentLanguageChanged -= value;
+                _languageDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Event to be invoked when an error occurs.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The state must be initialized.
+        /// </pre>
+        public static event EventHandler<ErrorOccurredEventArgs> ErrorOccurred
+        {
+            add
+            {
+                _errorDelegate = (ErrorCode reason, IntPtr userData) =>
+            {
+                string msg;
+                ErrorCode ret = VcMgrGetErrorMessage(out msg);
+                if (ret != ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get error message " + ret);
+                    msg = string.Empty;
+                }
+                ErrorOccurredEventArgs args = new ErrorOccurredEventArgs(reason, msg);
+                _errorOccurred?.Invoke(null, args);
+            };
+                ErrorCode error = VcMgrSetErrorCb(_errorDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add ErrorOccurred Failed with error " + error);
+                }
+
+                else
+                {
+                    _errorOccurred += value;
+                }
+            }
+
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetErrorCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove ErrorOccurred Failed with error " + error);
+                }
+
+                _errorOccurred -= value;
+                _errorDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Event to be called when dialog requests.
```

Comment:
Could you explain `dialog request` means?

Reply:
dialog request event is apllication (VC client) dialog msg.
apllication (VC client) can send to voice-control-manager lib and It is  related a component of UI or conversation panel.

will fixed "Called when an application (VC client) requests dialog."

Reply:
What kind of information was contained in dialog message?
I still can't understand, Why need this event? When application developer(VC client) send this event?
Who is receive this event?
How to react this event? how to handle this event? what action is expected?

Reply:
I confused because `XXXRequested` is a common pattern of naming. 
XXX should be a verb that represent some of action. 
But `Dialog` is not verb and does not represent some of action. 
So, I can't expect anything with this naming

Reply:
It seems to response message of commands
Dialog is just a one of displaying method
It can be shown on notification bar. 
I don't understand why you need to use `dialog` keyword

So, it better to rename, 
``` c#
        public static event EventHandler<ResponseMessageReceivedEventArgs> ResponseMessageReceived
```

Reply:
Or, If this message is not related with response of commands.
It is just a request to show a message on VoiceControlManager. 
So, it is better to rename
``` c#
  public static event EventHandler<MessageDisplayRequestedEventArgs> DisplayMessageRequested;
```

Reply:
Changed to ConversationRequestedEventArgs.